doc/languages-frameworks/python.md

```diff
@@ -744,6 +744,24 @@ Consider the packages `A` and `B` that depend on each other. When packaging `B`,
 a solution is to override package `A` not to depend on `B` as an input. The same
 should also be done when packaging `A`.
 
+### How to override a Python package from `configuration.nix`?
+
+If you need to change a package's attribute(s) from `configuration.nix` you could do:
+
+```nix
+  nixpkgs.config.packageOverrides = super: {
+    pythonPackages = super.pythonPackages // { bepasty-server = super.python27Packages.bepasty-server.overrideAttrs (oldAttrs: {
```

Comment:
not quite sure if we should recommend this, since this will not affect dependent python packages. A better approach is this, I believe:

```
pythonPackages = super.pythonPackages.override {
  overrides = self: super: {
    bepasty-server = super.bepasty-server.overrideAttrs { ... }
  };
}
```